app/actions/get-auth/get-regenerate/index.ts

```
@@ -0,0 +1,60 @@
+          ? authHeader.replace("Bearer ", "")
+          : null;
+        const newRefreshToken = response.headers.refreshtoken as string;
+
+        if (newAccessToken && newRefreshToken) {
+          useAuthStore.getState().setAccessToken(newAccessToken);
+          document.cookie = `refreshToken=${newRefreshToken}; path=/; Secure; SameSite=Strict;`;
+          return true;
+        }
+      }
+      return false;
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)